Integrations/integration-MISP.yml

```diff
@@ -417,6 +423,92 @@ script:
     }
 
 
+    var addMinutes = function(date, minutes) {
+                return new Date(date.getTime() + minutes*60000);
+    };
+
+
+    var getNowTime = function() {
+        var time = new Date();
+        time = addMinutes(time,time.getTimezoneOffset());
+        var month = "0" + (time.getMonth()+1);
+        var day = "0" + time.getDate();
+        var hours = "0" + time.getHours();
+        var minutes = "0" + time.getMinutes();
+        var seconds = "0" + time.getSeconds();
```

Comment:
what is the reason for this? looks like you assign value day /hours/ minutes/second but never use them.

Reply:
I do use it here:
var body = { 'Event': {
            'date': getNowTime(),
            'threat_level_id': '1',
            'published': args.published,
            'info': args.info,
            'analysis': '0',
            'distribution': '0',
            'Attribute': [currentArgs] } };

Reply:
Fixed